qa/smoke-test-rank-eval-with-mustache/src/test/java/org/elasticsearch/smoketest/SmokeMultipleTemplatesIT.java

```diff
@@ -82,27 +85,33 @@ public void testPrecisionAtRequest() throws IOException {
         List<RatedRequest> specifications = new ArrayList<>();
         Map<String, Object> ams_params = new HashMap<>();
         ams_params.put("querystring", "amsterdam");
-        RatedRequest amsterdamRequest = new RatedRequest("amsterdam_query", createRelevant("2", "3", "4", "5"), ams_params);
+        RatedRequest amsterdamRequest = new RatedRequest(
+                "amsterdam_query", createRelevant("2", "3", "4", "5"), ams_params, "match_template");
```

Comment:
Maybe make "match_template" a constant for this test since its used in several places.